src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1940,7 +1971,8 @@ void UsgsAstroLsSensorModel::losToEcf(
   // Define imaging ray (look vector) in camera space
    double cameraLook[3];
    createCameraLookVector(undistortedFocalPlaneX, undistortedFocalPlaneY,
-                          m_zDirection, m_focalLength + getValue(15, adj), cameraLook);
+                          m_zDirection, m_focalLength * (1 - getValue(15, adj) / m_halfSwath),

```

Comment:
This changes it back to the previous behavior. This also mean that adjusting the focal length adjustment results in the same adjustment as the other parameters.

Reply:
https://github.com/USGS-Astrogeology/usgscsm/blame/eb02f11be6e035bb4c0ab276637e05fc4d419788/src/UsgsAstroLsSensorModel.cpp#L1739